README.md

```diff
@@ -22,18 +20,20 @@ This application will be responsible for creating a User Interface that will all
 ### Requirements:
 
 - 1 React / JavaScript / Redux application.
-  - 1 Page for viewing the edited version of the Ruleset.
-  - OPTIONAL 1 Page for viewing the read-only version of the Ruleset.
+  - 1 Page for creating a new Ruleset.
+  - 1 Page for viewing the edited version of a Ruleset.
 
 - 1 Java / SpringBoot API
-  - 1 GET endpoint to retrieve the Ruleset and Rules.
-  - 1 POST endpoint for creating a new Ruleset and Rules.
-  - 1 PUT endpoint for updating the Ruleset and Rules.
+  - 1 GET endpoint to retrieve all Rulesets.
```

Comment:
Maybe add "(ID, Name, Date Created)" at the end.
"All Rulesets" makes it seem like it's the same endpoint as 28.